### DISCUSSION TITLE
Look for stable version

### DISCUSSION THREAD

**Discussion by monsat31** [geodesicsolutions-community/geocore-community#172] [Q&A] [answered]
Full script Addon
Hi frends i tried the bêta versions of This script and Always get some broken fonctionnalités such as addons not work properly
Or some php errors
Template errors 
Broken links

Does someone have thé good version pls

**Comment by vicos59** [comment]
You have already been provided with the latest stable code. Last place I see it was left is that your site had been hacked and files compromised due to a Wordpress exploit. A screenCap you posted earlier showed that you were not using the default template set. If you have not installed the optional template sets provided in the additional zip files, this could be the cause of your problems.

**Comment by monsat31** [comment]
thank you vicos59 i reinstall the code a gain and it work fine also the template work fine i think there were a bugs side server , also im looking for translate files sources for the 20 version also for addons and categories databases

**Comment by nicolaDorian** [comment]
hello guys, which is the   latest stable code and where ?  thanks

**Comment by vicos59** [comment]
> hello guys, which is the latest stable code and where ? thanks

https://github.com/vicos59/geocore-community/tree/main/RELEASES/Beta-5

I do not have access to this repository to upload a build, so I just put it in my copy of the repo. 

I would recommend you just stick with the default built-in template set and not mess with the optional ones --- at least until you get the system running the way you want it. Fewer variables introduced into the equation.

**Comment by nicolaDorian** [comment]
Thanks Vico

**Comment by jonyo** [comment, marked as answer]
The official beta 5 is now released!  Note that there have been a few things fixed after the one @vicos59 made so if you used one of his downloads he graciously built for folks, this release will have a few more things fixed in it.

See the [v20.0.0-beta.5 here](https://github.com/geodesicsolutions-community/geocore-community/releases/tag/v20.0.0-beta.5) - scroll down to the assets section.

**Reply by nicolaDorian** [reply]
thanks Jonio,
How to update from beta 3 to 5 ?   is this ?   https://geodesicsolutions.org/update-instructions/?product=GeoCore+CE+%28Community+Edition%29&version=20.0.0&method=&tplUpdate=export

**Reply by jonyo** [reply]
> thanks Jonio, How to update from beta 3 to 5 ? is this ? https://geodesicsolutions.org/update-instructions/?product=GeoCore+CE+%28Community+Edition%29&version=20.0.0&method=&tplUpdate=export

Yes that is it!
